index.html

```diff
@@ -4,11 +4,32 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link rel="stylesheet" href="body.css">
-
-    <h1>"Hello Microverse!"</h1>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <title>Frehiywot Nega Portifolio</title>
         
   </head>
   <body>
+    <section class="navigation">
+      <header class="mobile_header">
+        <div class="image_geometry_1"> </div>
+        <nav class="menu-button-selector">
+          <img src="image/ic_menu@2x.jpg" alt="menu" class="menu-button" />
+        </nav>
+      </header>
+    </section>
+    <section class="headline">
+    <h1>Frehiywot <br />Nega</h1>
+  </section>
+    <p class="main_pargraph">
+      Hello! I am a software developer! 
+      I can help you build a product, feature or website. 
+      Taka a look of my works.If you like what you see and have a project you need coded, 
+      don’t hesitate and contact me.
+    </p>
+    <a href="#start_collaboration" class="collaboration">Start collaboration</a>
+    <br>
+    <div class="image_geometry_2"></div> 
```

Comment:
Great job @frnega 🤝

- [ ] [Optional] It would have been better if the second section covered the rest of your project. This is because according to the project requirements we are building two sections for this project.